ydb/_topic_writer/topic_writer_asyncio_test.py

```diff
@@ -66,7 +75,7 @@ async def writer_and_stream(self, stream) -> WriterWithMockedStream:
             )
         )
 
-        writer = WriterAsyncIOStream(None)
+        writer = WriterAsyncIOStream(1, lambda: "foo-bar")
```

Comment:
1 is bad interval as default for test fixture, better is None (never), because many mock tests may not include update tokens as expected result. And test may flaky if CI slow and will failed when debug with manual debugger.

Reply:
good point, thank you